pkg/core/interop_system_test.go

```diff
@@ -826,6 +925,22 @@ func TestContractUpdate(t *testing.T) {
 	})
 }
 
+func TestContractDestroy(t *testing.T) {
+	v, cs, ic, bc := createVMAndContractState(t)
+	defer bc.Close()
+
+	v.LoadScriptWithHash(cs.Script, cs.Hash, smartcontract.All)
+	require.NoError(t, contractDestroy(ic)) // silent error when contract is missing
+	require.NoError(t, ic.DAO.PutContractState(cs))
+
+	v.Estack().PushVal("key")
+	v.Estack().PushVal("value")
+	require.NoError(t, storageGetContext(ic))
+	require.NoError(t, storagePut(ic))
+	require.NoError(t, contractDestroy(ic))
+	require.Error(t, storageGetContext(ic))
```

Comment:
```
storageDelete()
contractDestroy(cs.Hash)
```
?

Reply:
This was about checking that items were also deleted during `contractDestroy`.
I have added checks for that using DAO.